website/docs/docs/dbt-cloud-apis/user-tokens.md

```diff
@@ -8,7 +8,10 @@ pagination_next: "docs/dbt-cloud-apis/service-tokens"
 
 :::note Announcement
 
-The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). We recommend rotating your existing user tokens with PATs. There are no deprecation plans for user API tokens at this time; we will give ample notice when that timeline has been determined. 
+_User tokens will be deprecated on September 18th, 2024._
+
+The [user API tokens](#user-tokens) are being replaced by [account-scoped personal access tokens(PATs)](#account-scoped-personal-access-tokens). If you do not rotate your existing user tokens with PATs by September 18th, the services using the tokens will encounter errors. 
+
```

Comment:
Hiya @matthewshaver 

Great PR and no real comments or suggestions. I did notice the first link for API Scoped User Tokens didn't direct me to the relevant page / section. I've added the link for ref (/docs/dbt-cloud-apis/user-tokens#user-api-tokens).

Kind Regards
Natalie